fcc-test-suite-template-master/index.html

```diff
@@ -30,7 +30,7 @@ <h1><center>Here's a time line of Dr. César's life:</center></h1>
         <ul>
           <li><strong>1914</strong> - Born in Cresco, Iowa</li>
           <li>
-            <strong>1933</strong> - Leaves his family's farm to attend the
+            <strong>1933</strong> - Leaves his family farm to attend the
             University of Minnesota, thanks to a Depression era program known as
             the "National Youth Administration"
           </li>
```

Comment:
please make styling in separate folder

Reply:
- [ ] The `strong` tag is well used in this line, it's not only for style work but also highlights that this content is important so the screen reader will deliver this content to the users, so kindly Find any other piece of code that needs enhancement in your project and add your reviews to it 👍🏼